rllib/policy/policy.py

```diff
@@ -220,43 +220,45 @@ def compute_single_action(
             - state_outs: List of RNN state outputs, if any.
             - info: Dictionary of extra features, if any.
         """
-        # If policy works in normalized space, we should unsquash the action.
-        # Use value of config.normalize_actions, if None.
-        unsquash_actions = \
-            unsquash_actions if unsquash_actions is not None \
-            else self.config["normalize_actions"]
-        clip_actions = clip_actions if clip_actions is not None else \
-            self.config["clip_actions"]
-
-        prev_action_batch = None
-        prev_reward_batch = None
-        info_batch = None
+        # Build the input-dict used for the call to
+        # `self.compute_actions_from_input_dict()`.
+        if input_dict is None:
```

Comment:
for single action computation, we construct the input_dict here in policy.
but for multi-actions computation, we construct this input_dict in trainer.
should we keep this all in trainer instead? what do you think?